### ISSUE TITLE
Loss oscillation in pretraining on NTU60

### ISSUE THREAD

**Issue by ikramkourbane** [maoyunyao/MAMP#3]
Hi, thank you for this great work.

I tried the pretraining model on the **NTU60** following the setup instructions. The training **loss** is highly **oscillating**, would you please explain this behavior? the loss starts at 0.265 and goes to ~0.24 minimum, I wonder if this is normal? and what checkpoint should we take for the downstream task?

![image](https://github.com/maoyunyao/MAMP/assets/157490192/a31f5dd7-02e8-4679-a417-a9d790ac3d68)

Thank you.

**Comment by maoyunyao** [comment]
The last checkpoint (399).
